packages/components/src/icons/src/Icon.tsx

```diff
@@ -4,7 +4,7 @@ import { ResponsiveProp, StyledSystemProps, useResponsiveValue, useStyleProps }
 
 import { Box } from "../../box";
 
-export type IconSize = "2xs" | "xs" | "sm" | "md" | "lg" | "xl" | "inherit";
+export type IconSize = "inherit";
```

Comment:
Should we allow "md" and inherit, or only inherit? not sure if its weird that the only allowed size is inherit

Reply:
md would be placebo as anything not inherit would be md

Reply:
@alexasselin008